src/video/SDL_video.c

```diff
@@ -46,6 +46,14 @@
 #include "SDL_opengles2.h"
 #endif /* SDL_VIDEO_OPENGL_ES2 && !SDL_VIDEO_OPENGL */
 
+#if SDL_VIDEO_METALANGLE
+#include <MetalANGLE/GLES2/gl2.h>
```

Comment:
Are these headers needed here if the code is changed to keep `SDL_VIDEO_OPENGL_ES2` defined instead of making it mutually exclusive with metal-angle?

Reply:
Could be merged inside the `SDL_opengles2.h` file, with a `#if SDL_VIDEO_OPENGL_METALANGLE #else ...`